---
title: 57396 - ThrowingException
ms.custom: 
ms.date: 03/30/2017
ms.prod: .net-framework
ms.reviewer: 
ms.suite: 
ms.technology: dotnet-clr
ms.tgt_pltfrm: 
ms.topic: article
ms.assetid: 4f865de5-62c1-443c-9dcd-1675ca127334
caps.latest.revision: "3"
author: dotnet-bot
ms.author: dotnetcontent
manager: wpickett
ms.workload: dotnet
ms.openlocfilehash: 06378cb5d404830f210421091a1abaa0d7da2e3c
ms.sourcegitcommit: 16186c34a957fdd52e5db7294f291f7530ac9d24
ms.translationtype: MT
ms.contentlocale: it-IT
ms.lasthandoff: 12/22/2017
---
# <a name="57396---throwingexception"></a>57396 - ThrowingException
## <a name="properties"></a>Propriet√†  
  
|||  
|-|-|  
|ID|57396|  
|Parole chiave|Infrastruttura|  
|Livello|Avviso|  
|Canale|Microsoft-Windows-Application Server-Applications/Analytic|  
  
## <a name="description"></a>Descrizione  
 Questo evento viene generato in occasione di un'eccezione.  
  
## <a name="message"></a>Messaggio  
 Generazione di un'eccezione. Origine %1.  
  
## <a name="details"></a>Dettagli
